parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -298,6 +294,12 @@ if [ -f /var/run/reboot-required ]; then
     aptmarkWALinuxAgent unhold &
   fi
 else
+{{- if RunUnattendedUpgrades}}
```

Comment:
If we *are not* going to reboot, and we ran `unattended-upgrade`, then we need to start `kubelet` and `kubelet-monitor`, because they were not started earlier.